DESCRIPTION

```diff
@@ -1,15 +1,19 @@
 Package: moderndive
 Type: Package
 Title: Tidyverse-Friendly Introductory Linear Regression
-Version: 0.5.3.9000
+Version: 0.5.4
```

Comment:
No, sorry for the confusion. You want this version to be 0.5.3.9001. 

However, in the NEWS, you want the version number to be 0.5.4. 

The reasoning is that this minor version helps the developers keep track of where they are in the process. However, the NEWS file is more for users (and developers). So we'll let changes that we make in all of the minor versions accumulate for a while, and then when we're ready, we'll release version 0.5.4 of the package. 

In this case, all 10 of these PRs will accumulate before we jump to version 0.5.4. So all of the changes should pile up under the 0.5.4 in the NEWS file.